tests/backup/backup_test.go

```diff
@@ -3355,9 +3371,19 @@ var _ = Describe("{DeleteSharedBackup}", func() {
 
 		})
 		Step("Validating that backups are deleted from owner of backups", func() {
-			userBackups1, _ := GetAllBackupsAdmin()
-			log.InfoD("%v", userBackups1)
-			dash.VerifyFatal(len(userBackups1), 0, fmt.Sprintf("Validating that shared backups are deleted from owner of backup"))
+			adminBackups, _ := GetAllBackupsAdmin()
+			log.Infof("%v", adminBackups)
+			adminBackupsMap := make(map[string]bool)
+			for _, backup := range adminBackups {
+				adminBackupsMap[backup] = true
+			}
+			for _, name := range backupNames {
+				if adminBackupsMap[name] {
+					backupNotDeleted = name
+					break
```

Comment:
Log the names of the backups not deleted before breaking

Reply:
While executing the dash.VerifyFatal process, I am evaluating whether the backupNotDeleted variable contains any data. Regardless of the outcome, the backupNotDeleted information will be displayed there if I'm not wrong , and in the loop as soon I as found first backup which is not deleted I'm breaking the loop.

Reply:
In the dash.VerifyFatal procedure, I am checking if the backupNotDeleted variable is empty. In any case, the contents of backupNotDeleted will be printed there.